cli/cobra.go

```diff
@@ -359,6 +430,15 @@ Management Commands:
   {{rpad (decoratedName .) (add .NamePadding 1)}}{{.Short}}{{ if isPlugin .}} {{vendorAndVersion .}}{{ end}}
 {{- end}}
 
+{{- end}}
+{{- if hasOrchestratorSubCommands . }}
+
+Orchestration Commands:
```

Comment:
Could be more explicit by `Swarm Commands:`

Reply:
Yes, perhaps I should rename this; let me push a commit